interview-questions.md

```diff
@@ -7,44 +7,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: the foreign key would be on the student model. The name of the foreign key would be student_id becasue the foreign key is always the modelname_id. To fix this mistake we would put the foreign key with the student model.
 
-  Researched answer:
+  Researched answer: The foreign key always goes on the belongs_to side. To create the association between the tweo models by adding in the Cohort model file add "has_many" and add "belongs_to" in the Student model file.
```

Comment:
To update the model, you would need to run a migration so your schema has that as cohort_id as a column on the Student model.